bridgesync/processor.go

```diff
@@ -263,6 +265,9 @@ func (p *processor) Reorg(ctx context.Context, firstReorgedBlock uint64) error {
 	if err := tx.Commit(); err != nil {
```

Comment:
can we just not redeclare the err here?
if we do `if err = tx.Commit(); err != nil {`, will it not work as expected?